app/src/main/java/com/example/mymovieinfo/MainActivity.kt

```diff
@@ -24,12 +32,16 @@ class MainActivity : AppCompatActivity() {
 
         val btn_FindMyMovie = findViewById(R.id.btn_FindMyMovie) as Button
         btn_FindMyMovie.setOnClickListener {
-                // your code to perform when the user clicks on the button
+            // your code to perform when the user clicks on the button
 
             result.text = "the $name was released in $year"
 
 
-                //Toast.makeText(this@MainActivity, "You clicked me.", Toast.LENGTH_SHORT).show()
+            //Toast.makeText(this@MainActivity, "You clicked me.", Toast.LENGTH_SHORT).show()
         }
+        setContent {
+            viewModel.fetchCountries()
+            val movies by viewModel.movies.observeAsState(initial = emptyList())
```

Comment:
This looks like an incomplete feature.
Are you using these anywhere?  I don't see them used.
If not in use, remove them.  Keep the codebase as small as possible to do the job you want to do.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.